### PR TITLE
[Cookbook][Security] Replace deprecated csrf_provider service

### PR DESCRIPTION
**Pull request by xelaris**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.5+ |
| Fixed tickets |  |

The deprecation was introduced in symfony/symfony@d4bb5f4.

### CODE REVIEW

File: cookbook/security/csrf_in_login_form.rst [hunk 1]
```diff
@@ -60,12 +60,17 @@ provider available in the Form component:
...
-                        'csrf_provider' => 'form.csrf_provider',
+                        'csrf_provider' => 'security.csrf.token_manager',
                     )
                 )
             )
         ));
 
+.. versionadded:: 2.4
+    The ``security.csrf.token_manager`` service was introduced in Symfony 2.4.
+    Prior to Symfony 2.4 you can use the ``form.csrf_provider`` service
```
Comment (WouterJ) [inline cookbook/security/csrf_in_login_form.rst]: "Prior to Symfony 2.4, you can use [...]"

**Comment by xelaris** [comment]
Fixed. Thanks @WouterJ

**Comment by WouterJ** [comment]
:+1: Will merge later this weekend :)

**Comment by WouterJ** [comment]
As there are just 25 minutes left before the weekend is over here, I have to merge this PR :) Thanks Alexander for your recent activity on the docs!

**Comment by WouterJ** [comment]
Btw, I've removed the versionadded directive as we don't have 2.4 versionaddeds in the 2.6 branch (the 2.5 branch is no longer maintained)

**Comment by WouterJ** [comment]
gg

**Pull request closed without merging**